core/controllers/editor_test.py

```diff
@@ -613,11 +613,10 @@ def test_exploration_download_handler_for_default_exploration(self):
         # Check download to zip file.
         # Download to zip file using download handler.
         download_url = '/createhandler/download/%s' % exp_id
-        response = self.get_response(download_url,
-                                     expected_content_type='text/plain')
+        response = self.get_custom_response(
+            download_url, expected_content_type='text/plain')
```

Comment:
You don't need to specify the argname for expected_content_type, since it has no default value. 

The linter should catch this. Maybe file a bug?

Reply:
done

Reply:
Opened #6067